### Annotation

Gene model prediction was performed by combining a lifting approach using [liftoff](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8289374/) (v1.6.3) and a genome-guided approach using transcriptomics data with [HISAT2](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7605509/) (v2.2.1), [StringTie](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4643835/) (2.1.7) and [Transdecoder](https://github.com/TransDecoder/TransDecoder/wiki) (v5.7.0). Post-processing of gff3 files and filtering were performed using [AGAT](https://github.com/NBISweden/AGAT) and [gffread](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7222033/) (v0.11.7). 

For the gene lifting, gene models of hexaploid wheat line Chinese Spring, *Ae tauschii*, and *Triticum monoccocum* accession TA29932 were independently transferred using liftoff. For the genome-guided approach, publicly available RNA-Seq data of 12 representative *Ae. umbellulata* accessions and the RNA-Seq data of two bulks representing *Ae. umbellulata* leaf tissues were used. All the RNA-Seq data were mapped individually against the reference sequence using HISAT2 and the transcripts were assembled using StringTie and merged into a single gtf file. The [Transdecoder.LongOrfs](https://github.com/TransDecoder/TransDecoder/blob/master/TransDecoder.LongOrfs) script was used to identify open reading frames (ORF) of at least 100 amino acids from the merged gtf file. The predicted protein sequences were compared to the UniProt and Pfam databases using BLASTP and [hmmer3](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3125773/). The [Transdecoder.Predict](https://github.com/TransDecoder/TransDecoder/blob/master/TransDecoder.Predict) script was used with the BLASTP and hmmer results to select the best translation per transcript. Finally, the annotation gff3 file was computed using the perl script [*cdna_alignment_orf_to_genome_orf.pl*](https://github.com/TransDecoder/TransDecoder/blob/master/util/cdna_alignment_orf_to_genome_orf.pl).

All the output gff files from the lifting and genome-guided approaches were merged and filtered to retain transcripts with start and stop codons. In total, 36,268 gene models were predicted for which the putative functional annotations were assigned using a protein comparison against the UniProt database using [DIAMOND](https://www.nature.com/articles/nmeth.3176). PFAM domain signatures and GO were assigned using [InterproScan](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1160203/).

Repeated sequences were called with the Repeat Detector, which is part of the [Ensembl Genomes repeat feature pipelines](http://plants.ensembl.org/info/genome/annotation/repeat_features.html). Repeats length: 3514124750 - Repeats content: 82.7%
