### Assembly

Main assembly consisted of 133.2x of PACBIO coverage (6,139 average read size), and was assembled using MECAT and the resulting sequence was polished using QUIVER. The primary transcripts from the V3 release of Populus trichocarpa (var. Nisqually) were used to identify misjoins in the Populus assembly. Misjoins were characterized by an abrupt change in the Paspalum linkage group. A total of 12 breaks were made to the assembly.

Scaffolds were then oriented, ordered, and joined together using gene synteny. Significant telomeric sequence was properly oriented in the assembly. A total of 81 joins were applied to the broken assembly to form the final assembly consisting of 19 chromosomes. 99.2% of the assembled sequence is contained in the chromosomes.

Adjacent alternative haplotypes were identified on the joined contig set. Althap regions were collapsed using the longest common substring between the two haplotypes. A total of 21 adjacent altHaps were collapsed. A set of 676 finished BAC clones were then aligned to the chromosomes in an attempt to patch remaining gaps. One gap on Chr07 was patched, resulting in the addition of 16,801 bases.

Heterozygous snp/indel phasing errors were corrected using the 133.2x raw PACBIO data. A total of 53,017 (2.03% of the 2,614,201 Heterozygous snps/indels) were corrected. Additionally, homozygous SNPs and INDELs were corrected in the release sequence using 100x of Illumina reads (2x150, 500bp insert, library ID IPFY).

