## Wheat assemblies

Ensembl Plants hosts the [latest wheat assembly](http://plants.ensembl.org/Wheat) from the IWGSC (RefSeq v1.0), including:

- The IWGSC RefSeq v1.1 gene annotation, with links to [wheat-expression.com](http://www.wheat-expression.com) and [KnetMiner](https://knetminer.rothamsted.ac.uk/Triticum_aestivum/)
- [5 UK wheat cultivars](/Triticum_aestivum/Info/Strains?db=core) : Cadenza, Claire, Paragon, Robigus, Weebill
- Alignment of 98,270 high confidence genes from the TGACv1 annotation.
- Axiom 35K, 820K SNP arrays from [CerealsDB](https://www.cerealsdb.uk.net/cerealgenomics/CerealsDB/index.php), including QTL links in selected cases and Linkage Disequilibrium display. See QTL example [here](https://plants.ensembl.org/Triticum_aestivum/Variation/Explore?r=5A:438266461-438267461;v=BA00617086;vdb=variation;vf=264386).
- EMS-induced mutations from sequenced TILLING populations (Kronos and Cadenza).
- Inter-Homeologous Variants (IHVs) between the A, B and D genome components.
- Chromosome specific KASP markers were added from the Nottingham BBSRC Wheat Research Centre.
- Whole genome alignments to rice, brachypodium and barley.
- Assembly-to-assembly mapping and gene ID mapping to the previous TGAC v1 assembly, archived at [eg37-plants.ensembl.org](http://eg37-plants.ensembl.org)
- Polyploid view enabled, allowing users to view alignments among multiple wheat components [simultaneously](http://plants.ensembl.org/Triticum_aestivum/Location/MultiPolyploid?db=core;g=TraesCS7A02G200000;r=7A:162572423-162579126;r1=7D:161387881-161394584:1;r2=7B:123652787-123659490:1;s1=Triticum_aestivum--7D;s2=Triticum_aestivum--7B;t=TraesCS7A02G200000.1)
- Durum wheat 35K, 90K, 820K and TaBW280K variants
