### References

1.  [Shifting the limits in wheat research and breeding using a fully
    annotated reference
    genome](http://dx.doi.org/10.1126/science.aar7191).\
    Rudi Appels, Kellye Eversole, Catherine Feuillet, Beat Keller, Jane
    Rogers, Nils Stein\.... Hana imkov, Ian Small, Manuel Spannagl,
    David Swarbreck, Cristobal Uauy. 2018. Science. 361
2.  [Uncovering hidden variation in polyploid
    wheat](http://dx.doi.org/10.1073/pnas.1619268114).\
    Ksenia V. Krasileva, Hans A. Vasquez-Gross, Tyson Howell, Paul
    Bailey, Francine Paraiso, Leah Clissold, James Simmonds, Ricardo H.
    Ramirez-Gonzalez et al. . 2016. PNAS. 114:E913E921.
3.  [An improved assembly and annotation of the allohexaploid wheat
    genome identifies complete families of agronomic genes and provides
    genomic evidence for chromosomal
    translocations](http://dx.doi.org/http://www.genome.org/cgi/doi/10.1101/gr.217117.116.).\
    Bernardo J. Clavijo, Luca Venturini,Christian Schudoma, Gonzalo
    Garcia Accinelli, Gemy Kaithakottil, Jonathan Wright, Philippa
    Borrill, George Kettleborough, Darren Heavens, Helen Chapman et
    al. 2017. Genome Research.
4.  [Ultra-Fast Next Generation Human Genome Sequencing Data Processing
    Using DRAGENTM Bio-IT Processor for Precision
    Medicine](http://dx.doi.org/https://doi.org/10.4236/ojgen.2017.71002).\
    Goyal, A., Kwon, H.J., Lee, K., Garg, R., Yun, S.Y. et al. 2017.
    Open Journal of Genetics. 7:9-19.
5.  [CerealsDB 3.0: expansion of resources and data
    integration](http://dx.doi.org/10.1186/s12859-016-1139-X).\
    Wilkinson PA, Winfield MO, Barker GL,Tyrrell S, Bian X, Allen AM,
    Burridge A, Coghill JA, Waterfall C, Caccamo M et al. 2016. BMC
    Bioinformatics. 17:256.
6.  [PolyMarker: A fast polyploid primer design
    pipeline](http://dx.doi.org/doi:10.1093/bioinformatics/btv069).\
    Ricardo H. Ramirez-Gonzalez, Cristobal Uauy, Mario Caccamo. 2015.
    Bioinformatics. 31:2038-2039.
7.  [Developing integrated crop knowledge networks to advance candidate
    gene
    discovery](http://dx.doi.org/https://doi.org/10.1016/j.atg.2016.10.003).\
    Keywan Hassani-Pak, Martin Castellote, Maria Esch, Matthew Hindle,
    Artem Lysenkoa Jan Taubert and Christopher Rawlings. 2016. Applied &
    Translational Genomics. 11:18-26.
8.  [The transcriptional landscape of hexaploid wheat across tissues and
    cultivars](http://dx.doi.org/doi:10.1126/science.aar6089).\
    Ramrez-Gonzlez RH, Borrill P, Lang D, Harrington SA, Brinton J,
    Venturini L, Davey M, Jacobs J, van Ex F, Pasha A et al. 2018.
    Science. 361
9.  [expVIP: a customisable RNA-seq data analysis and visualisation
    platform](http://dx.doi.org/doi:10.1104/pp.15.01667).\
    Philippa Borrill, Ricardo Ramirez-Gonzalez, and Cristobal
    Uauy. 2016. Plant Physiology .

10. [Rapid identification of homozygosity and site of wild relative introgressions in wheat through 
    chromosome‐specific KASP genotyping assays](https://onlinelibrary.wiley.com/doi/full/10.1111/pbi.13241).\
    Surbhi Grewal, Stella Hubbart‐Edwards, Caiyun Yang, Urmila Devi, Lauren Baker, Jack Heath, Stephen Ashling, Duncan Scholefield, 
    Caroline Howells, Jermaine Yarde, Peter Isaac, Ian P. King and Julie King.
    2016. Plant Biotechnology Journal. 18:743–755.

11. [Integrating genomic resources to present full gene and putative promoter capture probe sets for bread wheat](https://academic.oup.com/gigascience/article/8/4/giz018/5304888).\
	Gardiner LJ, Brabbs T, Akhunov A, Jordan K, Budak H et al. 2019. Gigascience 8(4): giz018

12. [Efficient genome-wide detection and cataloging of EMS-induced mutations using next-generation sequencing and exome capture](http://www.plantcell.org/content/26/4/1382).\
	Henry IM, Nagalakshmi U, Lieberman MC, Ngo KJ, Krasileva KV et al. 2014. Plant Cell 26:1382–1397

