### References

1. Alaux, M. et al. (2018). Linking the International Wheat Genome Sequencing Consortium bread wheat reference genome sequence to wheat genetic and phenomic data. *Genome Biology*, 19:111.

2. Appels, R., Eversole, K., Feuillet, C., et al. (IWGSC) (2018). Shifting the limits in wheat research and breeding using a fully annotated reference genome. *Science*, 361:eaar7191.

3. Bushnell, B., Rood, J., & Singer, E. (2017). BBMerge - Accurate paired shotgun read merging via overlap. *PLoS ONE*, 12(10):e0185056. https://doi.org/10.1371/journal.pone.0185056

4. Clavijo, B. J. et al. (2017). W2RAP: a pipeline for high quality, robust assemblies of large complex genomes from short read data. *bioRxiv* [Preprint]. https://doi.org/10.1101/110999

5. Danecek, P., Bonfield, J. K., Liddle, J., et al. (2021). Twelve years of SAMtools and BCFtools. *GigaScience*, 10(2), giab008. https://doi.org/10.1093/gigascience/giab008

6. Li, H. (2018). Minimap2: pairwise alignment for nucleotide sequences. *Bioinformatics*, 34(18), 3094–3100. https://doi.org/10.1093/bioinformatics/bty191

7. Monat, C., Padmarasu, S., Lux, T., et al. (2019). TRITEX: chromosome-scale sequence assembly of Triticeae genomes with open-source tools. *Genome Biology*, 20, 284. https://doi.org/10.1186/s13059-019-1899-5

8. Quinlan, A. R., & Hall, I. M. (2010). BEDTools: a flexible suite of utilities for comparing genomic features. *Bioinformatics*, 26(6), 841–842. https://doi.org/10.1093/bioinformatics/btq033

9. Shumate, A., & Salzberg, S. L. (2021). Liftoff: accurate mapping of gene annotations. *Bioinformatics*, 37(12), 1639–1643. https://doi.org/10.1093/bioinformatics/btaa1016

10. Walkowiak, S., Gao, L., Monat, C., et al. (2020). Multiple wheat genomes reveal global variation in modern breeding. *Nature*, 588, 277–283. https://doi.org/10.1038/s41586-020-2961-x
