### Assembly

Nearly 341 Gb of Illumina reads were generated for DH00/086, utilizing insert sizes that ranged from short insert paired-end libraries to larger mate-pair libraries. An initial assembly was produced using SOAPdenovo, gap-filled with GapCloser, and further scaffolded with SSPACE. To improve its contiguity, 6.07Gb of Pacific Biosystems long-reads were produced for further gap-filling and scaffolding. To anchor the assembly onto pseudomolecules representing the 10 haploid chromosomes, a population of 74 doubled haploid individuals were low-depth resequenced to produce a genetic map. The genetic map data were used to identify chimeric contigs or scaffolds in the genome assembly resulting from mis-assembly. The resulting scaffolds were then mapped onto the optical map contigs and conflicting data manually edited. Only 6.2% of the genome assembly could not be assigned to chromosomal locations in the linkage map. 
